devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -77,10 +78,16 @@ def on() {
 }
 
 def refresh() {
-    zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig() + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
+    zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig(0,300) + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
```

Comment:
I get your reasoning from a code organization angle. I just want to be careful not a alter the DH in ways that don't affect the intent of the ticket, especially since I'm not as familiar with ZigBee and our utility class and any pitfalls around returning a list of commands to issue to the device.  
Is it equivalent to say the following as the body of refresh():

```
List zigbeeRefresh = zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh()
zigbeeRefresh += zigbee.onOffConfig(0,300) + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
return zigbeeRefresh
```